tests/acceptance/patients-test.js

```diff
@@ -1,179 +1,150 @@
-    andThen(() => {
-      assert.equal(find('.clickable').length, 0, 'No patient is listed');
-    });
-  });
-});
-
-test('Adding a new patient record', function(assert) {
-  runWithPouchDump('default', function() {
-    authenticateUser();
-    visit('/patients/edit/new');
-    andThen(function() {
-      assert.equal(currentURL(), '/patients/edit/new');
-    });
-
-    fillIn('.test-first-name input', 'John');
-    fillIn('.test-last-name input', 'Doe');
-    click('.panel-footer button:contains(Add)');
-    waitToAppear('.message');
-    andThen(function() {
-      assert.equal(find('.message').text(), 'Patient Saved', 'Patient record has been saved');
-      assert.equal(find('.modal-body').text().trim(), 'The patient record for John Doe has been saved.', 'Record has been saved');
-    });
-    click('button:contains(Close)');
-    waitToAppear('.patient-summary');
-
-    andThen(function() {
-      findWithAssert('.patient-summary');
-    });
-    andThen(function() {
-      findWithAssert('#general');
-    });
-
-  });
-});
-
-function testSimpleReportForm(reportName) {
-  test(`View reports tab | ${reportName} shows start and end dates`, function(assert) {
-    runWithPouchDump('default', function() {
-      authenticateUser();
-      visit('/patients/reports');
-      select('[data-test-selector="select-report-type"] select', reportName);
-
-      andThen(function() {
-        let reportStartDate = find('[data-test-selector="select-report-start-date"]');
-        let reportEndDate = find('[data-test-selector="select-report-end-date"]');
-        assert.equal(reportStartDate.length, 1, 'Report start date select is visible');
-        assert.equal(reportEndDate.length, 1, 'Report end date select is visible');
-        let reportType = find('[data-test-selector="select-report-type"] select');
-        assert.equal(reportType.find(':selected').text().trim(), reportName, `${reportName} option selected`);
-      });
-    });
-  });
-}
-
-function testExportReportName(reportName) {
-  test(`View reports tab | Export reports name for ${reportName} shows report name, start and end dates`, (assert) => {
-    runWithPouchDump('default', () => {
-      authenticateUser();
-      visit('/patients/reports');
-      select('[data-test-selector="select-report-type"] select', reportName);
-
-      andThen(() => {
-        assert.equal(currentURL(), '/patients/reports');
-      });
-
-      fillIn('[data-test-selector="select-report-start-date"] input', '12/11/2016');
-      fillIn('[data-test-selector="select-report-end-date"] input', '12/31/2016');
-
-      click('button:contains(Generate Report)');
-      waitToAppear('.panel-title');
-
-      andThen(() => {
-        let exportReportButton = findWithAssert('a:contains(Export Report)');
-        assert.equal($(exportReportButton).attr('download'), `${reportName} Report 12/11/2016 - 12/31/2016.csv`);
-      });
-    });
-  });
-}
+ import Ember from 'ember';
```

Comment:
@jkleinsc I had used eslint to fix the errors of the particular file.May be because of that the whole file appears to be reformatted.I am going to replace the existing file with the previous one so that you can easily merge it.